torch_xla_py/xla_model.py

```diff
@@ -141,28 +141,41 @@ def __init__(self, smooth_factor=0.8):
     self._smooth_factor = smooth_factor
     self._start_time = time.time()
     self._partial_time = self._start_time
-    self._count = 0
-    self._rate = 0.0
+    self._partial_count = 0
```

Comment:
Maybe use float here to avoid accidentally dividing 2 ints on line 178. Not sure if this file is guaranteed to run only on python3 or if python2 is also possible